atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1829,6 +1829,20 @@ gfx::Size WebContents::GetSizeForNewRenderView(content::WebContents* wc) const {
   return gfx::Size();
 }
 
+void WebContents::SetOwnerWindow(NativeWindow* owner_window) {
+  CommonWebContentsDelegate::SetOwnerWindow(owner_window);
+#if BUILDFLAG(ENABLE_OSR)
```

Comment:
Can you move this to `CommonWebContentsDelegate::SetOwnerWindow` ? The overrides in `api::WebContents` seems redundant.

Reply:
Sure, I just felt like this made more sense, rather than moving these to `CommonWebContentsDelegate` which has nothing to do currently with OSR.

Reply:
`CommonWebContentsDelegate` owns the actual `content::WebContents` through brightray and acts as the delegate for `api::WebContents` , `api::WebContents` is just layer exposed to users, any implementation details should be move there if its not affecting user api directly.